### PR TITLE
feat(PYME-3909): Add system column in each row

### PR DESCRIPTION
**Pull request by PaneroEduardo**
* Change `template.xml`
* Add `SISTEMA` field with the constant value `TES`

### CODE REVIEW

**Review by PACriado — APPROVED** [review]

File: reports/request_type_status_report/templates/xlsx/template.xlsx
Comment (ruben-garciad) [inline reports/request_type_status_report/templates/xlsx/template.xlsx:1]: why did you delete this template?

**Review by cx01754 — APPROVED** [review]

File: reports/request_type_status_report/templates/xlsx/template.xlsx
Comment (PaneroEduardo) [reply reports/request_type_status_report/templates/xlsx/template.xlsx:1]: this template is not deleted.

**Review by ruben-garciad — APPROVED** [review]
